### PR TITLE
Update Bls private keys

### PR DESCRIPTION
**Pull request by rlan35**

### CODE REVIEW

File: consensus/view_change.go [hunk 1]
```diff
@@ -371,7 +371,7 @@ func (consensus *Consensus) onViewChange(msg *msg_pb.Message) {
 		consensus.consensusTimeout[timeoutViewChange].Stop()
 		consensus.consensusTimeout[timeoutConsensus].Start()
 		utils.GetLogger().Debug("new leader start consensus timeout and stop view change timeout", "viewID", consensus.viewID, "block", consensus.blockNum, "viewChangingID", consensus.mode.ViewID())
-		utils.GetLogger().Debug("I am the new leader", "myKey", consensus.PubKey.GetHexString()[:20], "viewID", consensus.viewID, "block", consensus.blockNum)
+		utils.GetLogger().Debug("I am the new leader", "myKey", consensus.PubKey.SerializeToHexStr()[:20], "viewID", consensus.viewID, "block", consensus.blockNum)
```
Comment (mikedoan) [inline consensus/view_change.go]: remove [:20]

Comment (mikedoan) [reply consensus/view_change.go]: and other similar places

File: internal/genesis/foundational.go [hunk 2]
```diff
@@ -2,113 +2,113 @@ package genesis
...
-	{Address: "0x144B2Fd168147311f749B0f9573664676C333e2A", Public: "0x144B2Fd168147311f749B0f9573664676C333e2A", BLSKey: "2bae3aeb4653657c1b30e4204e8c093bbab8e3ba93b421325dc0a3f60e53549c"},
-	{Address: "0x22117D26611161b1b1f4EBB06C441aeeA102261c", Public: "0x22117D26611161b1b1f4EBB06C441aeeA102261c", BLSKey: "588ec71b9999a4007663432e8e3fa50aae972ebe3bad89148f42e588cf101701"},
-	{Address: "0x24A8cD56bABef297F1C7234F830362466d01ff5d", Public: "0x24A8cD56bABef297F1C7234F830362466d01ff5d", BLSKey: "50a40637369e7d9d704f063c2c818a53e628d087b8d5a57a9661f4017c5ba0"},
-	{Address: "0x25347d09373B2644191f1DC4beDEFEBE26a5b2d1", Public: "0x25347d09373B2644191f1DC4beDEFEBE26a5b2d1", BLSKey: "72d41ddbfadf9b74f8ec4d6bf2c911b90325bf5bf6620ddd891c78fcf1b358d7"},
-	{Address: "0x25441821ecA41DEc79578aAB866d3627A2e9BB9f", Public: "0x25441821ecA41DEc79578aAB866d3627A2e9BB9f", BLSKey: "570dce63c362a8222fd564516538c84c81fdbc3d8dea0bd11129e0080c9d6486"},
+	{Address: "0x04c3636dF766ad2d3E74424c016842f5704FAE3A", BlsPriKey: "ff889b96e38934c08ea158ce32fb94ec605180a4f665ed378aea9b9ac1c39320"},
```
Comment (mikedoan) [inline internal/genesis/foundational.go]: we don't want to expose blsprivate key to public? - we should save here https://github.com/harmony-one/secret_keys -

File: internal/genesis/genesis.go [hunk 3]
```diff
@@ -63,488 +62,488 @@ var DeployedContractAddress = crypto.CreateAddress(crypto.PubkeyToAddress(Genesi
...
-	{Address: "0x2c2a172192a84aF1d37F8793c596b9b91b517ede", Public: "0x2c2a172192a84aF1d37F8793c596b9b91b517ede", BLSKey: "1d314f61823af4c64639e8e283ffceb141449789621d10d78189c32a60d55bf9"},
-	{Address: "0x2e9e70D46290A33F3F01251587AB09C84FCE7cb7", Public: "0x2e9e70D46290A33F3F01251587AB09C84FCE7cb7", BLSKey: "f04d9c79df912210daa3198350c8524ff1192f6d5c6630438078443270ef48f"},
-	{Address: "0x2eE12a8225e5e811e11264D123c4E180F0797be7", Public: "0x2eE12a8225e5e811e11264D123c4E180F0797be7", BLSKey: "6ddc97eb5c8d7b64fda8e75c6bbe9f407bbafb84d1b73ad57b5eae5644557559"},
-	{Address: "0x3076E2D79053c964966204Fd1A0aF770ef2538D8", Public: "0x3076E2D79053c964966204Fd1A0aF770ef2538D8", BLSKey: "3dd53a752ae9551e6b5047bcf17d0abb5a8161d254fb3fd7a19e52a7ab1bface"},
-	{Address: "0x31335ed908287cdF7f9416cF748d145eE7913B34", Public: "0x31335ed908287cdF7f9416cF748d145eE7913B34", BLSKey: "377c13bcca1add4dc74d79942610e651b9f9d41fe91e3064ad7469818fb06d27"},
+	{Address: "0x2E258FeeC9Fad9d243B946FDB490c19434E78a26", BlsPriKey: "c79d77feafe263ed104c8a3dd3a78f4c24cb15f23bf378d78536b0fed2f81d11"},
```
Comment (mikedoan) [inline internal/genesis/genesis.go]: same - we don't want to expose this to public

File: consensus/view_change.go (hunk 1 above)
Comment (rlan35) [reply consensus/view_change.go]: This is the old code, I bet the purpose is to save log space.

**Pull request merged**